### PR TITLE
Log version on startup

### PR DESCRIPTION
**Pull request by edmundnoble**
Exactly what it says on the tin. Logs in the exact same place and with the exact same message as specified in the ticket.

#qz-3689 Done

### CODE REVIEW

File: web/src/main/scala/quasar/server/Server.scala [hunk 1]
```diff
@@ -183,6 +183,7 @@ object Server {
 
   def safeMain(args: Vector[String]): Task[Unit] = {
     logErrors(for {
+      _ <- Task.delay(println(s"Quasar v${quasar.build.BuildInfo.version}")).liftM[MainErrT]
```
Comment (jedesah) [inline web/src/main/scala/quasar/server/Server.scala]: For consistency, might as well use `stdout` as below

**Comment by jedesah** [comment]
Weird, why didn't Travis run? Also, looks like AppVeyor failed, I don't know if that's typical these days

**Comment by edmundnoble** [comment]
Appveyor isn't working right now, there are plan problems. Not sure about Travis.

**Comment by jedesah** [comment]
Okay, LGTM, will merge

**Comment by jedesah** [comment]
@edmundnoble Can you update to latest master? Looks like I can't merge as is with the merge script

**Comment by jedesah** [comment]
@edmundnoble If Travis passes, feel free to merge. I'll be away from my computer for the rest of the day, I'll merge if you haven't already tonight

**Pull request merged**